src/utils/Reducers/CategoriesReducer.js

```diff
@@ -1,14 +1,12 @@
-import { ACTION_TYPE_LOADING, ACTION_TYPE_SUCCESS, ACTION_TYPE_ERROR } from "../actionTypes";
-
 export const CategoriesReducer = (categories, action) => {
     switch(action.type) {
-        case ACTION_TYPE_LOADING : 
+        case "ACTION_TYPE_LOADING" : 
             return {...categories, categoryStatus:"loading"}
 
-        case ACTION_TYPE_SUCCESS :
+        case "ACTION_TYPE_SUCCESS" :
             return {...categories,  categoryStatus:"success", categoryData:action.payload}
            
-        case ACTION_TYPE_ERROR :
+        case "ACTION_TYPE_ERROR" :
             return {...categories, categoryStatus:"error", categoryData:action.payload}
             
```

Comment:
BrandReducer and CategoriesReducer are very similar. Instead of creating separate reducers, you can create a common/shared reducer as well.

Reply:
yes will implement a custom hook for that.